src/main/java/org/folio/service/orders/CompositePoLineValidationService.java

```diff
@@ -291,4 +292,34 @@ private List<Error> validatePackagePoLine(CompositePoLine compPOL) {
 
     return convertErrorCodesToErrors(compPOL, errors);
   }
+
+  public List<Error> validateForBinadryActive(CompositePoLine poLine) {
+    List<Error> errors = new ArrayList<>();
+    if (poLine.getDetails() != null && poLine.getDetails().getIsBindaryActive()) {
+      validateOrderFormatForBindaryActive(poLine, errors);
+      validateCreateInventoryForBindary(poLine, errors);
```

Comment:
we also should add validation for this case:
"AND Receiving workflow is set to Independent order and receipt quantity"
 We have flag 'checkinItems' for this.
checkinItems == true -> Independent flow
checkinItems == false -> Synchronized flow

Reply:
Thanks for this finding, implemented